src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs

```diff
@@ -419,27 +417,27 @@ public ImageSource Icon
                         bitmap = Properties.Resources.ZeroTouchNodeReferenceIcon;
                         break;
 
-                    case ReferenceType.External when MimeMapping.GetMimeMapping(DependencyInfo.Name).Contains("image"):
+                    case ReferenceType.External when MimeUtility.GetMimeMapping(DependencyInfo.Name).Contains("image"):
```

Comment:
btw, this code looks odd.. `when` is used for range based switch, couldn't we just use equality here?